calcom/synclinear.com
# Sync milestones <> cycles

# Summary

- 1:1 sync of Linear cycles and GitHub milestones

## Test Plan

- Create Linear cycle in synced team. Ensure milestone is created with same title, description, `open` state (unless past `endsAt`), and link in footer.
- Edit Linear cycle title, description, and state. Ensure milestone is synced.
- Create GH milestone in synced repo. Ensure Linear cycle is created with same title, description, and end date.
- Edit milestone title and description. Ensure cycle reflects changes.
- Mark milestone as `completed`. Ensure cycle `endsAt` date is set to today.

TODO:
- [x] Auto-update webhooks to include milestone/project events
- [x] Synced ticket added to synced cycle <> synced milestone assigned to synced issue

## Related Issues

Closes #5
